generation/um/minidumpapiset/generate.rsp

```diff
@@ -0,0 +1,19 @@
+@../../settings.rsp
+@../../remap.rsp
+--file
+um-minidumpapiset.h
+--output
+../../../sources/Interop/Windows/um/minidumpapiset
+--test-output
+../../../tests/Interop/Windows/um/minidumpapiset
+--traverse
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.20348.0/um/minidumpapiset.h
+--exclude
+MiniDumpWriteDump
```

Comment:
Uh I would not exclude MiniDumpWriteDump, instead I could suggest to route ``_MINIDUMP_CALLBACK_INFORMATION``, ``_MINIDUMP_CALLBACK_INPUT``, ``_MINIDUMP_THREAD_CALLBACK``, ``_MINIDUMP_THREAD_EX_CALLBACK`` which I think it takes in as a pointer to ``IntPtr`` or ``nint``?